src/loc/cs-cz.ts

```diff
@@ -0,0 +1,421 @@
+    "FolderFilterBoxPlaceholder": "Filtrovat složky podle názvu",
+    "FolderExplorerLoading": "Načítání složek...",
+    "FolderExplorerNoItems": "Tato složka neobsahuje žádné podsložky.",
+    "NewFolderIncorrectSymbolsError": "Zadejte název, který neobsahuje žádný z těchto znaků: \" * : < > ? / \ |.",
+    "SomethingWentWrong": "Něco se pokazilo",
+    "SelectedLabel": "Vybráno",
+    "SelectIcon": "Vybrat ikonu",
+
+    "TreeViewExpandTitle": "Rozbalit položku",
+    "TreeViewCollapseTitle": "Sbalit položku",
+
+    "CollectionDataEmptyFields": "Pro kolekci nebyla definována žádná pole.",
+    "CollectionDataEmptyValue": "Kolekce neobsahuje data.",
+    "CollectionAddRowButtonLabel": "Přidat data do kolekce",
+    "CollectionDeleteRowButtonLabel": "Odstranit aktuální řádek",
+    "CollectionSaveAndAddButtonLabel": "Přidat a uložit",
+    "CollectionDataItemShowErrorsLabel": "Zobrazit chyby řádku",
+    "CollectionDataItemFieldRequiredLabel": "Pole je povinné.",
+    "CollectionDataItemMissingFields": "Kolekce polí je prázdná!",
+    "InvalidUrlError": "Zapsaná URL adresa je neplatná",
+    "CollectionDataSearch": "Hledat",
+    "CollectionDataMinimumUsersDefaultMessage": "Minmální počet uživatelů k výběru je {0}",
+
+    "MyTeamsLoadingMessage": "načítání vašich týmů",
+    "MyTeamsMessageError": "Při načítání vašich týmu došlo k chybě. Opakujte akci později, nebo proveďte obnovení stránky v prohlížeči (F5)",
+    "MyTeamsNoTeamsMessage": "Nemáte žádné týmy",
+    "MyTeamsTeamChannelPublicMessage": "Veřejné kanály",
+    "MyTeamsTeamChannelTypeMessage": "Privátní kanály",
+    "TeamChannelPickerFontIconFavoriteText": "Oblíbené",
+    "TeamChannelPickerFontIconPrivateChannelTitle": "Privátní kanál",
+    "TeamChannelPickerSugestionHeaderText": "Doporučené kanály",
+    "TeamPickerButtonRemoveTitle": "odstranit",
+    "TeamPickernoResultsFoundText": "Týmy nenalezeny",
+    "TeamPickerSugestionsHeaderText": "Doporučené týmy",
+    "TeamsChannelPickerButtonRemoveTitle": "odstranit",
+    "TeamsChannelPickerNoresultsFoundText": "Kanály nenalezeny",
+
+    "ViewMore": "Zobrazit více",
+
+    "DynamicFormLoading": "Načítání...",
+    "DynamicFormPleaseWait": "Počkejte...",
+    "DynamicFormRequiredErrorMessage": "Toto pole nemůže zůstat prázdné.",
+    "DynamicFormTermPanelTitle": "Vybrat výraz",
+    "DynamicFormEnterURLPlaceholder": "Zadejte URL",
+    "DynamicFormEnterDescriptionPlaceholder": "Alternativní popis",
+    "DynamicFormDialogValidationErrorTitle": "Chyba validace",
+    "DynamicFormDialogValidationErrorMessage": "Formulář obsahuje chyby validace. Před uložením je potřeba tyto chyby odstranit.",
+    "DynamicFormNumberValueMustBeGreaterThan": "Hodnota musí být větší než {0}",
+    "DynamicFormNumberValueMustBeBetween": "Hodnota musí být mezi {0} a {1}",
+    "DynamicFormNumberValueMustBeLowerThan": "Hodnota musí být menší než {0}",
+    "customDisplayName": "Použít toto umístění:",
```

Comment:
There're missing location props that should be translated:

* `DynamicFormChooseFileLabel`
* `DynamicFormChooseFileButtonText`
* `DynamicFormRequiredFileMessage`

You can find example [here](https://github.com/pnp/sp-dev-fx-controls-react/blob/dev/src/loc/en-us.ts#L391)